### ISSUE TITLE
Make it server side

### ISSUE THREAD

**Issue by janat08** [ziarno7/refresh-sound#2]
Client will not always refresh, and dev may refresh the page himself.

**Comment by ziarno** [comment]
PRs are welcome ;)

**Comment by janat08** [comment]
const play = require('audio-play');
const load = require('audio-loader');
 
load('./sample.mp3').then(play);

tell it to load on server

**Comment by janat08** [comment]
load takes encoded string, and is node.js

**Comment by janat08** [comment]
Ideally it would also play during compilation errors.

**Comment by janat08** [comment]
Detects compilation error. Although this is client side, which I suppose is good enough.
https://github.com/stubailo/meteor-dev-error-overlay/blob/master/dev-error-overlay.js
```
setInterval(checkErrorState, 500);

// In development mode a DDP/Websocket disconnection is likely a sign of a
// broken server. In this case we re-check the error state immediately instead
// of waiting on average 250ms (worst case 500ms).
Tracker.autorun(() => {
  if (! Meteor.status().connected) {
    checkErrorState();
  }
});

function checkErrorState() {
  HTTP.get('/', (err, res) => {
    // Don't ring the alert in case of a server disconnection. We only want to
    // alert the developer in case there is a server-side exception breaking the
    // app. If the server is not responding we assume that the developer stopped
    // it on purpose (for instance to test offline mode) and we won't do
    // anything.
    if (err) {
      return;
    }

    const isMeteorApp = res.content.indexOf('__meteor_runtime_config__') !== -1;

    if (!overlay && !isMeteorApp) {
      startErrorReport();
      runAlerts();
    } else if (overlay && !isMeteorApp && errorPage !== res.content) {
      refreshErrorReport();
      runAlerts();
    } else if (overlay && isMeteorApp) {
      stopErrorReport();
    }
    errorPage = res.content;
  });
}
```

**Comment by janat08** [comment]
I tried running server play sound and arcane exception begun popping up, but then I'm on linux/1.6.rc.0 so perhaps if you package and wrap it up nicely then it won't be so much of a baby.
